src/main/java/de/pandemieduell/model/Card.java

```diff
@@ -0,0 +1,13 @@
+package de.pandemieduell.model;
+
+import java.util.List;
+
+public interface Card {
+  String getName();
+
+  String getDescription();
+
+  List<Action> getActions();
+
+  int getNumberOfTickets(int round, WorldState state);
```

Comment:
What are tickets?

Reply:
We are missing a good name for that.
The idea is, that this number is used for calculating the probability for the card when drawing new cards for the players

Reply:
Ah, I see.